packages/sanity/src/core/releases/components/BundleMenuButton/BundleMenuButton.tsx

```diff
@@ -80,7 +80,8 @@ export const BundleMenuButton = ({disabled, bundle, documentCount}: BundleMenuBu
             disabled={bundleMenuDisabled || isPerformingOperation}
             icon={isPerformingOperation ? Spinner : EllipsisHorizontalIcon}
             mode="bleed"
-            padding={2}
+            style={{padding: 2}}
```

Comment:
Do we strictly need this? I think we could use the `size="small"` so it matches with the existing buttons.

Reply:
- [ ] use default ui-component button and ignore padding: 2 cases